include/uzfs_rebuilding.h

```diff
@@ -56,10 +56,13 @@ int uzfs_get_nonoverlapping_ondisk_blks(zvol_state_t *zv, uint64_t offset,
     uint64_t len, blk_metadata_t *incoming_md, void **list);
 int
 uzfs_zvol_create_snaprebuild_clone(zvol_state_t *zv,
```

Comment:
snaprebuild seems to be confusing, and, all the names are not in sync.
can we have function names as:
- uzfs_zvol_create_internal_clone
- uzfs_zvol_destroy_internal_clone
- uzfs_zvol_release_internal_clone
with description for each of these APIs?

Reply:
Done.